src/python/grapl-template-generator/grapl_template_generator/main.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+import typer
+
+from typing import Union, Literal
+
+from grapl_template_generator.python_http_service.create_python_http_service import create_python_http_service
+from grapl_template_generator.python_http_service.create_python_http_service_args import CreatePythonHttpServiceArgs
+from grapl_template_generator.rust_grpc_service.create_rust_grpc_service import create_rust_grpc_service
+from grapl_template_generator.rust_grpc_service.create_rust_grpc_service_args import CreateRustGrpcServiceArgs
+
+"""
+Generators for:
+* Asynchronous/Synchronous services in Rust and Python
+"""
+
+app = typer.Typer()
+
+@app.command(name="py-http")
+def generate_python_http_server(
+        project_name: str,
+        pants_version: str = typer.Argument("2.4.0"),
+        pants_python_interpreter_constraints: str = typer.Argument("CPython==3.7.*"),
+        pants_black_version_constraint: str = typer.Argument("black==20.8b1"),
+        pants_isort_version_constraint: str = typer.Argument("isort==5.6.4"),
+        pants_mypy_version_constraint: str = typer.Argument("mypy==0.8"),
+        lambda_handler: str = "lambda_handler",
+) -> None:
+    args = CreatePythonHttpServiceArgs(
+        project_name=project_name,
+        pants_version=pants_version,
+        pants_python_interpreter_constraints=pants_python_interpreter_constraints,
+        pants_black_version_constraint=pants_black_version_constraint,
+        pants_isort_version_constraint=pants_isort_version_constraint,
+        pants_mypy_version_constraint=pants_mypy_version_constraint,
+        lambda_handler=lambda_handler,
+    )
+    create_python_http_service(args)
+
+
+@app.command(name="rust-grpc")
+def generate_python_http_server(
+        package_name: str,
+        cargo_version: str = typer.Argument("1.52.0"),
+        rustc_channel: str = typer.Argument("stable"),
```

Comment:
Since this is automation code, it would be nice for it to automatically set these values (and the Python ones above) to whatever we're currently using, rather than force us to remember to update things in multiple places.

(if indeed these values really need to be set at all; we probably shouldn't have rust sub-projects building with a different version of Rust than everything else, for instance.)